src/cls/rbd/cls_rbd_types.h

```diff
@@ -133,6 +133,7 @@ inline void decode(MirrorImageStatusState &state, bufferlist::iterator& it)
 
 struct MirrorImageStatus {
   MirrorImageStatus() {}
+  ~MirrorImageStatus() {}
```

Comment:
Destructor should be after constructors -- but this still doesn't make sense as to why Clang is requiring an explicit destructor within the function ```librbd::mirror::GetStatusRequest<>::send```